### DISCUSSION TITLE
[API PRODUCTION ERROR] getFeaturedListingsByShop is returning 500 error

### DISCUSSION THREAD

**Discussion by BoredGameGeek** [etsy/open-api#745] [Q&A] [answered]
As of today and for at least the past 5 days, the [getFeaturedListingsByShop](https://developers.etsy.com/documentation/reference#operation/getFeaturedListingsByShop) endpoint has a problem.

Replication case:
- Example shop: https://www.etsy.com/shop/darkcycleclothing (shop id `5503963`) - go to that page and note that there ARE featured items.
- Send a `GET` to https://openapi.etsy.com/v3/application/shops/5503963/listings/active?limit=5&offset=0  (include `x-api-key` header with valid API key)
  - Expected: valid JSON response
  - Actual: Valid JSON response ✔️
    - ![image](https://user-images.githubusercontent.com/78888562/207784596-7a54e7e6-4c44-4034-a4cd-621dd0b0c3ed.png)

- Send a `GET` to https://openapi.etsy.com/v3/application/shops/5503963/listings/featured?limit=5&offset=0 (same url as above, but for featured listings) (include `x-api-key` header with valid API key)
  - Expected: valid JSON response
  - Actual: `500 Internal Server Error` ❌
    - ![image](https://user-images.githubusercontent.com/78888562/207784655-194a8829-e8af-4e39-a341-37108aa26680.png)

**Comment by etsyachristensen** [comment, marked as answer]
Thanks @BoredGameGeek. This is logged to be fixed.

**Reply by etsyachristensen** [reply]
This has been fixed @BoredGameGeek . Sorry for the trouble!
